app/controllers/api/account.php

```diff
@@ -2268,17 +2270,16 @@
     ->label('abuse-key', 'userId:{userId}')
     ->inject('request')
     ->inject('response')
-    ->inject('phone')
     ->inject('user')
     ->inject('dbForProject')
     ->inject('audits')
     ->inject('events')
     ->inject('usage')
     ->inject('messaging')
-    ->action(function (Request $request, Response $response, Phone $phone, Document $user, Database $dbForProject, Audit $audits, Event $events, Stats $usage, EventPhone $messaging) {
+    ->action(function (Request $request, Response $response, Document $user, Database $dbForProject, Audit $audits, Event $events, Stats $usage, EventPhone $messaging) {
 
-        if (empty(App::getEnv('_APP_PHONE_PROVIDER'))) {
-            throw new Exception(Exception::GENERAL_PHONE_DISABLED, 'Phone provider not configured');
+        if (empty(App::getEnv('_APP_SMS_PROVIDER'))) {
+            throw new Exception('Phone provider not configured', 503, Exception::GENERAL_PHONE_DISABLED);
```

Comment:
```suggestion
            throw new Exception(Exception::GENERAL_PHONE_DISABLED);
```
The exception class has the following signature 
`Exception($type, $message, $code)`